### DISCUSSION TITLE
Hotkey to add a separator line in text pane

### DISCUSSION THREAD

**Discussion by advice2020** [geany/geany#4027] [Ideas]
A couple programs I have used in the past allow the ability to setup a hotkey that automatically types in a separator which I have found very convenient.  User basically chooses the symbol used and the length of how many to type in to simulate a separator line.
`##################################`
`=================================`
I was wondering if any developers would be interested in adding something like this into Geany in the future.
I think you can probably use something like snippets but I do not believe you can set this up to be one simple hotkey that just activates this.  If anyone knows of a way please let me know, but for now I will keep this in the Ideas section to hopefully be added in the future.
I would mainly use in plain text files so not an issue, but maybe in the case of Geany because it can deal with multiple languages, maybe more beneficial if the symbol is based on the comment out symbol used by that specific language.
Probably not something that anyone would be interested in but figured I would mention because I found this very useful in other applications.

Thank You

**Comment by elextr** [comment]
It could be added to one of the plugins that add additional features (check the most likely).

**Comment by eht16** [comment]
I think using a snippet for this is roughly equivalent in choosing a symbol and the amount of repittions or just type the symbol of choice, mark it and duplicate the selection (`Ctrl-D`) as often as you like.
